pipelines/dbx-pipeline.yml

```diff
@@ -12,227 +12,283 @@ pool:
-      inputs:
-        script: |
-          cd "wheel"
-          python setup.py sdist bdist_wheel
-
-
-    - task: PythonScript@0
-      displayName: "upload wheel to DBFS"
-      inputs:
-        scriptSource: 'filePath' 
-        scriptPath: pipelineScripts/upload_wheel_to_dbfs.py
-      env:
-        DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
-        DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
-        DefaultWorkingDirectory: $(System.DefaultWorkingDirectory)
-
-
-    - task: PythonScript@0
-      displayName: "uninstall wheel from cluster"
-      inputs:
-        scriptSource: 'filepath'
-        scriptPath: pipelineScripts/uninstall_wheel_from_cluster.py
-      env:
-        DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
-        DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
-        DBX-WORKSPACE-URL: $(DbxWorkspaceUrl)
-     
-    - task: PythonScript@0
-      displayName: "restart cluster"
-      inputs:
-        scriptSource: 'filepath'
-        scriptPath: pipelineScripts/restart_cluster.py
-      env:
-        DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
-        DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
-  
-    - task: PythonScript@0
-      displayName: "install wheel"
-      inputs:
-        scriptSource: 'filepath'
-        scriptPath: pipelineScripts/install_wheel.py
-      env:
-        DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
-        DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
-  
-    - task: PythonScript@0
-      displayName: "check wheel status"
-      inputs:
-        scriptSource: 'filepath'
-        scriptPath: pipelineScripts/check_wheel_status.py
-      env:
-        DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
-        DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
+        - task: PythonScript@0
+          displayName: "restart cluster"
+          inputs:
+            scriptSource: 'filepath'
+            scriptPath: pipelineScripts/restart_cluster.py
+          env:
+            DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
+            DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
+            DBRKS_CLUSTER_ID: $(DBRKS_CLUSTER_ID')
+            DBRKS_SUBSCRIPTION_ID: '$(SubscriptionID)'
+            DBRKS_INSTANCE: '$(DBXInstance)'
+            DBRKS_RESOURCE_GROUP: '$(ResourceGroup)'
+            DBRKS_WORKSPACE_NAME: '$(WorkspaceName)'
+      
+        - task: PythonScript@0
+          displayName: "check wheel status"
+          inputs:
+            scriptSource: 'filepath'
+            scriptPath: pipelineScripts/check_wheel_status.py
+          env:
+            DBRKS_BEARER_TOKEN: $(DBRKS_BEARER_TOKEN)
+            DBRKS_MANAGEMENT_TOKEN: $(DBRKS_MANAGEMENT_TOKEN)
+            DBRKS_CLUSTER_ID: $(DBRKS_CLUSTER_ID')
+            DBRKS_SUBSCRIPTION_ID: '$(SubscriptionID)'
+            DBRKS_INSTANCE: '$(DBXInstance)'
+            DBRKS_RESOURCE_GROUP: '$(ResourceGroup)'
+            DBRKS_WORKSPACE_NAME: '$(WorkspaceName)'
+            
+    - job: delete_cluster
```

Comment:
Do we want to add comments here so people know not to delete the cluster by default? As this will undo previous steps in the pipeline, including uploading the wheel?